api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -178,4 +179,18 @@ public void validate_shouldPassForAValidAllergy() {
 		
 		assertFalse(errors.hasErrors());
 	}
+
+	@Test
+	public void valid_shouldRejectNumericReactionValue() throws Exception {
+		Allergy allergy = new Allergy();
+		AllergyReaction reaction = new AllergyReaction();
+		String nonCoded = "sometext";
```

Comment:
Are you sure that this is a numeric value?

Reply:
It is not, @HerbertYiga when I put any numeric value.........like "sometext4" and I trigger System println........... It surely...shows that it will validate......so do u hv to put it like "sometext4" dere